multinet/api.py

```diff
@@ -43,6 +43,17 @@ def get_workspace_tables(workspace: str, type: TableType = "all") -> Any:  # noq
     return util.stream(tables)
 
 
+@bp.route("/workspaces/<workspace>/tables", methods=["POST"])
+@use_kwargs({"table": fields.Str()})
+@swag_from("swagger/workspace_aql_tables.yaml")
+def create_table_from_aql(workspace: str, table: str) -> Any:
```

Comment:
I prefer shorter function names when possible. This should be `create_aql_table`.